charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -24,6 +27,9 @@ data:
     -XX:+HeapDumpOnOutOfMemoryError
     -XX:+ExitOnOutOfMemoryError
     -Djdk.attach.allowAttachSelf=true
+  {{- range $configValue := .Values.additionalJVMConfig }}
```

Comment:
There are a lot of recommended JVM options missing which can lead to perf issues. Please copy the config from https://trino.io/docs/current/installation/deployment.html#jvm-config